presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -2054,6 +2054,30 @@ public void testJoinWithConstantExpression()
     }
```

Comment:
The description is quite complex. I've seen a reference to GH issues in some commits already.

Will change that, though.